docs/en/ingest-management/commands.asciidoc

```diff
@@ -292,6 +298,13 @@ verified. The content is encrypted, but the certificate is not verified.
 +
 We strongly recommend that you use a secure connection.
 
+`--tag <string>`::
+A comma-separated list of tags to apply to {fleet}-managed {agent}s. You can
+use these tags to filter the list of agents in {fleet}.
++
+NOTE: Currently, there is no way to remove or edit existing tags. To change the
+tags, you must unenroll the {agent}, then re-enroll it using new tags.
```

Comment:
Thanks for noting this! This will be changing in 8.4 with the implementation of a more fleshed out tagging UI. Tracked here: https://github.com/elastic/ingest-dev/issues/1043